src/test/java/org/medibloc/panacea/GRPCTest.java

```diff
@@ -0,0 +1,154 @@
+package org.medibloc.panacea;
+
+import com.google.protobuf.ByteString;
+import cosmos.auth.v1beta1.BaseAccount;
+import cosmos.bank.v1beta1.MsgSend;
+import cosmos.base.abci.v1beta1.TxResponse;
+import cosmos.base.v1beta1.Coin;
+import cosmos.tx.v1beta1.*;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.medibloc.panacea.domain.Coins;
+import org.medibloc.panacea.domain.Transactions;
+import org.medibloc.panacea.utils.CryptoUtils;
+import panacea.aol.v2.MsgAddRecord;
+import tendermint.p2p.DefaultNodeInfo;
+
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.util.Arrays;
+import java.util.List;
+
+public class GRPCTest {
+    // owner mnemonic
+    String ownerMnemonic = "genuine key outside escape oval unhappy mansion cricket practice quarter purchase picnic layer bicycle stem soup column creek convince obey rather rice there alcohol";
+    String ownerAddress = "panacea1gtx6lmnjg6ykvv07ruyxamth6yuhgcvmhg3pqz";
+
+    // to mnemonic
+    String toMnemonic = "margin river antenna nest drama combine crystal clarify route country genuine acoustic hold ocean image fox flame invite scrub chalk destroy resource item final";
+    String toAddress = "panacea1rwxd29u3cmsuh9gmmrzlujl2e5lc8p33flpz2e";
```

Comment:
Well. It's a tricky part. 

Because Panacea Github repos are public, it's recommended to not put mnemonics and corresponding addresses together in source codes. If someone wants to hack a certain address and he/she can infer its mnemonic from source codes, it could be risky. But yeah, although the probability is super low, it's better to be safe than sorry. 

In this test code, I think we can remove `ownerAddress` and `toAddress`. Instead, we can derive `ownerAddress` by:
```java
this.ownerWallet.getAddress();
```

Reply:
OK. I removed address in code!